File: C/String/program14.c
//Accept the string from user and display the how many number of spaces contain in the string

#include<stdio.h>

int CountSpaces(char str[])
{
    int iCnt = 0;

    while(*str != '\0')
    {
        if(*str == ' ')
        {
            iCnt++;
        }
        str++;
    }
    return iCnt;
}

int main()
{
    char Arr[20];
    int iRet = 0;

    printf("Enter string: \n");
    scanf("%[^'\n']s",Arr);

    iRet = CountSpaces(Arr);
    printf("Number of spaces are :%d",iRet);

    return 0;
}

File: Java/String/program14.java
//Accept the string from user and display the how many number of spaces contain in the string

import java.util.*;

class Stringx
{
    public String str;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        str = sobj.nextLine();
    }

    public int CountSpaces()
    {
        char Arr[] = str.toCharArray();

        int i = 0;
        int iCnt = 0;

        for(i = 0; i < Arr.length; i++)
        {
            if(Arr[i] == ' ')
            {
                iCnt++;
            }
        }
        return iCnt;
    }
}

class program14
{
    public static void main(String arg[])
    {
        int iRet = 0;

        Stringx obj = new Stringx();

        obj.Accept();

        iRet = obj.CountSpaces();
        System.out.println("Number of spaces are : " +iRet);
    }
}